js/post-select/components/browse-filters.js

```diff
@@ -81,7 +81,12 @@ PostBrowseFilters.defaultProps = {
 };
 
 PostBrowseFilters.propTypes = {
-	value: PropTypes.object,
+	value: PropTypes.objectOf(
+		PropTypes.oneOf( [
```

Comment:
should this be `oneOfType`?

Reply:
Nope. This is correct as it is.

Reply:
but these are types not enums? e.g. `oneOf(['red','green'])`